server/opendp_apps/dataverses/views/dataverse_user_view.py

```diff
@@ -28,79 +27,72 @@ def create(self, request):
         # ----------------------------------
         # print(f"data: {request.data}")
         user_id = request.data.get('user')
-        try:
-            dataverse_user = DataverseUser.objects.get(object_id=request.data.get('user'))
-        except DataverseUser.DoesNotExist:
-            return Response({'success': False,
-                             'message': f'DataverseUser not found with object_id {user_id}'},
-                            status=status.HTTP_400_BAD_REQUEST)
-
+        handoff_id = request.data.get('dv_handoff')
         try:
             handoff = DataverseHandoff.objects.get(object_id=request.data.get('dv_handoff'))
```

Comment:
Could this be moved to the DataverseUserSerializer? e.g. 
```python
    def validate_dv_handoff(self, value):
        """
        Retrieve the DataverseHandoff based on the object_id
        """
        try:
            dv_handoff = DataverseHandoff.objects.get(object_id=str(value))
        except DataverseHandoff.DoesNotExist:
            raise serializers.ValidationError(f"DataverseHandoff does not exist for object_id: {value}")

        return dv_handoff
```

Reply:
I've added a function called `get_object_or_error_response` which essentially does this